backend/app/cmd/server.go

```diff
@@ -1058,8 +1058,8 @@ func (s *ServerCommand) makeSSLConfig() (config api.SSLConfig, err error) {
 	return config, err
 }
 
-func (s *ServerCommand) makeAuthenticator(ctx context.Context, ds *service.DataStore, avas avatar.Store, admns admin.Store, authRefreshCache *authRefreshCache) (*auth.Service, error) {
-	authenticator := auth.NewService(auth.Opts{
+func (s *ServerCommand) makeAuthenticator(ds *service.DataStore, avas avatar.Store, admns admin.Store, authRefreshCache *authRefreshCache) *auth.Service {
```

Comment:
this one probably need a comment indicating whay the returned auth service is not really made as it is not including any providers. Maybe it should be called prepAuthenticator instead of make, not sure